Change per SW551825

adangster opened the pull request:
https://w3.rchland.ibm.com/projects/bestquest/?defect=SW551825

Extra change that is not related to FRU_PATH or avs_slave attribute are part of the newer common_mrw change. should be dont care for this change.
DCM0:
     chip0--avs0--C14--vdd-0
     chip0--avs1-- vdn-vcs0 (located on system planar P0)
     chip1--avs0--C14--vdd-1
     chip1--avs1--vio-vcs0 (located on system planar P0)
DCM1:
     chip0--avs0--C23--vdd-0
     chip0--avs1-- vdn-vcs-1 (located on system planar P0)
     chip1--avs0--C23--vdd-1
     chip1--avs1--vio-vcs-1 (located on system planar P0)

adangster committed:
Change per
https://w3.rchland.ibm.com/projects/bestquest/?defect=SW551825
DCM0:
     chip0--avs0--C14--vdd-0
     chip0--avs1-- vdn-vcs0 (located on system planar P0)
     chip1--avs0--C14--vdd-1
     chip1--avs1--vio-vcs0 (located on system planar P0)
DCM1:
     chip0--avs0--C23--vdd-0
     chip0--avs1-- vdn-vcs-1 (located on system planar P0)
     chip1--avs0--C23--vdd-1
     chip1--avs1--vio-vcs-1 (located on system planar P0)

op-jenkins commented:
Can one of the admins verify this patch?

dcrowell77 commented:
retest this please

chenduibm commented:
the bus connection has an extra layer here, the "vdd-0" is not needed: <bus_id>proc_socket-0/godel-0/power10-0/avs0-master => vdd-vrm-connector-0/vdd-vrm-0/vdd-0/avs-slave</bus_id>

when you look at everest it looks like this: <bus_id>proc_socket-0/godel-0/power10-0/avs0-master => VRM-vdd-C62-conn/norgay-vrm-vdd-0/avs-slave</bus_id>

adangster commented:
Okay, i am not familiar with Rainier .. but after reading your comment .. the end result for the FRU_PATH call out will the same, Let me move the avs_slave to one layer higher.



> the bus connection has an extra layer here, the "vdd-0" is not needed: <bus_id>proc_socket-0/godel-0/power10-0/avs0-master => vdd-vrm-connector-0/vdd-vrm-0/vdd-0/avs-slave</bus_id>
> 
> when you look at everest it looks like this: <bus_id>proc_socket-0/godel-0/power10-0/avs0-master => VRM-vdd-C62-conn/norgay-vrm-vdd-0/avs-slave</bus_id>

adangster committed:
https://w3.rchland.ibm.com/projects/bestquest/?defect=SW551825
update per
chenduibm commented 6 hours ago
the bus connection has an extra layer here, the "vdd-0" is not needed: <bus_id>proc_socket-0/godel-0/power10-0/avs0-master => vdd-vrm-connector-0/vdd-vrm-0/vdd-0/avs-slave</bus_id>

adangster commented:
@chenduibm please see the new change above if that fit your requirement. Thanks.

chenduibm commented:
that fixed one issue but now I see another here: <bus_id>proc_socket-0/godel-0/power10-0/avs1-master => vdn-vcs-0/avs-slave</bus_id>

that is missing the bus connector above the "vdn-vcs-0". it looks like all your bus connections associated with avs1-master has that missing bus connector

adangster committed:
Pick up new common_mrw _fw1020 branch commit #948 per Nick B request (SW552995 )
No change to system mrw.

dcrowell77 commented:
please test

dcrowell77 commented:
retest this please

adangster commented:
> that fixed one issue but now I see another here: <bus_id>proc_socket-0/godel-0/power10-0/avs1-master => vdn-vcs-0/avs-slave</bus_id>
> 
> that is missing the bus connector above the "vdn-vcs-0". it looks like all your bus connections associated with avs1-master has that missing bus connector

This is expected ... already discussed this via slack ..these are does not have connector as they are on the motherboard / not removeable
